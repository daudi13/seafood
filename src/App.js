import { BrowserRouter, Route, Routes } from 'react-router-dom';
import './App.css';
import DetailsPage from './components/Pages/DetailsPage';
import Homepage from './components/Pages/Homepage';

function App() {
  return (
    <BrowserRouter>
      <Routes>
        <Route path="/" element={<Homepage />} />
        <Route path="/detailsPage" element={<DetailsPage />} />
      </Routes>
    </BrowserRouter>
  );
}

export default App;
