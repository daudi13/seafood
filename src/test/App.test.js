/* eslint linebreak-style: ["error", "windows"] */
import { render } from '@testing-library/react';
import React from 'react';
import { Provider } from 'react-redux';
import App from '../App';
import store from '../redux/configureStore';

describe('render App correctly', () => {
  test('render', () => {
    const tree = render(
      <React.StrictMode>
        <Provider store={store}>
          <App />
        </Provider>
      </React.StrictMode>,
    );
    expect(tree).toMatchSnapshot();
  });
});
