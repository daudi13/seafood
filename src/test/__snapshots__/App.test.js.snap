// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render App correctly render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header>
        <nav
          class="nav"
        >
          <img
            alt="fish"
            class="nav-icons"
            src="./images/polar-bear.png"
          />
          <h2
            class="nav-title"
          >
            SaveEarth
          </h2>
          <div
            class="nav-search"
          >
            <img
              alt="mic"
              class="nav-icons mic"
              src="./images/mic.png"
            />
            <img
              alt="mic"
              class="nav-icons"
              src="./images/settings-cog.png"
            />
          </div>
        </nav>
        <img
          alt="fishing boat"
          class="header-img"
          src="https://images.unsplash.com/photo-1544551763-77ef2d0cfc6c?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470&q=80"
        />
        <div
          class="image-cover"
        />
      </header>
      <h4>
        Animal species
      </h4>
      <main
        class="main"
      />
    </div>
  </body>,
  "container": <div>
    <header>
      <nav
        class="nav"
      >
        <img
          alt="fish"
          class="nav-icons"
          src="./images/polar-bear.png"
        />
        <h2
          class="nav-title"
        >
          SaveEarth
        </h2>
        <div
          class="nav-search"
        >
          <img
            alt="mic"
            class="nav-icons mic"
            src="./images/mic.png"
          />
          <img
            alt="mic"
            class="nav-icons"
            src="./images/settings-cog.png"
          />
        </div>
      </nav>
      <img
        alt="fishing boat"
        class="header-img"
        src="https://images.unsplash.com/photo-1544551763-77ef2d0cfc6c?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470&q=80"
      />
      <div
        class="image-cover"
      />
    </header>
    <h4>
      Animal species
    </h4>
    <main
      class="main"
    />
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
